pylint_pytest/checkers/fixture.py

```diff
@@ -135,8 +135,22 @@ def visit_module(self, node):
 
                 FixtureChecker._pytest_fixtures = fixture_collector.fixtures
 
-                if (ret != pytest.ExitCode.OK or fixture_collector.errors) and is_test_module:
-                    self.add_message('cannot-enumerate-pytest-fixtures', node=node)
+                legitimate_failure_paths = set(
+                    collection_report.nodeid
+                    for collection_report in fixture_collector.errors
+                    if any(
+                        fnmatch.fnmatch(
+                            Path(collection_report.nodeid).name, pattern,
+                        )
+                        for pattern in FILE_NAME_PATTERNS
+                    )
+                )
+                if (ret != pytest.ExitCode.OK or legitimate_failure_paths) and is_test_module:
+                    self.add_message(
+                        'cannot-enumerate-pytest-fixtures',
+                        args=' '.join(legitimate_failure_paths | {node.file}),
```

Comment:
I found that there's a potential duplication from this line: `legitimate_failure_paths` can contain relative paths, but the `node.file` is in absolute path. In my env it's throwing out pytest warning with duplicated paths:
```
sandbox/test_parent.py:1:0: F6401: pylint-pytest plugin cannot enumerate and collect pytest fixtures. Please run `pytest --fixtures --collect-only sandbox/test_parent.py /Users/reverbc/Workspace/pylint-pytest/sandbox/test_parent.py` and resolve any potential syntax error or package dependency issues (cannot-enumerate-pytest-fixtures)
```
Where the `sandbox/test_parent.py` and `/Users/reverbc/Workspace/pylint-pytest/sandbox/test_parent.py` are pointing to the same file.

Can you help to unify the path strings, maybe with `str(Path(...).absolute())` for both `collection_report.nodeid` and `node.file`?